/graphslam_scan_matcher_node:
    ros__parameters:
        # Methods: NDT_OMP, FAST_GICP, GICP
        registration_method: "NDT_OMP"
        
        base_frame_id: "base_link"
        displacement: 0.1
        max_scan_accumulate_num: 50
        use_imu: true
        
        # General settings
        max_iteration: 128
        omp_num_thread: 6
        transformation_epsilon: 0.01

        # NDT_OMP
        step_size: 0.1
        ndt_resolution: 2.0

        # FAST_GICP
        max_correspondence_distance: 2.0
        correspondence_randomness: 20

        # GICP
        correspondence_distance: 2.5
        euclidean_fitness_epsilon: 1e-05
        correspondence_randomness: 20
        max_optimizer_iteration: 100
        use_reciprocal_correspondences: false
